plugins/glean/src/index.ts

```diff
@@ -0,0 +1,79 @@
+import {
+  AnalyticsApi,
+  AnalyticsEvent,
+  AnyApiFactory,
+  ConfigApi,
+  analyticsApiRef,
+  configApiRef,
+  createApiFactory,
+} from '@backstage/core-plugin-api';
+
+import Glean from '@mozilla/glean/web';
+import GleanMetrics from '@mozilla/glean/metrics';
+import { JsonObject } from '@backstage/types';
+import { create } from './metrics/backstage';
+
+interface DebugConfig extends JsonObject {
+  logging?: boolean,
+  tag?: string,
+}
+
+export class GleanAnalytics implements AnalyticsApi {
+  private constructor(appId: string, enabled: boolean, config: object, debug?: DebugConfig) {
+    if (debug) {
+      Glean.setLogPings(!!debug.logging);
+      // set the debug tag if it is defined
+      debug.tag && Glean.setDebugViewTag(debug.tag);
+    }
+    Glean.initialize(appId, enabled, config);
+  }
+
```

Comment:
Curious: noticed how the type for `..., config: object, ...` is `object`? I see below we are using `ConfigApi` type for the `config` argument. Are they off different type? if so, is it possible to use a more concrete type vs `object` 🤔

Reply:
They are different yes. the `config` object being passed in here is handed to the `Glean` API. However, that uses a specific type. I will update it to be more specific.